diff --git a/src/image-block/view/index.ts b/src/image-block/view/index.ts
index 2109c65..b4caef7 100644
--- a/src/image-block/view/index.ts
+++ b/src/image-block/view/index.ts
@@ -14,7 +14,7 @@ export const imageBlockView = $view(
   imageBlockSchema.node,
   (ctx): NodeViewConstructor => {
     return (initialNode, view, getPos) => {
-      const src = ref(initialNode.attrs.src)
+      const src = ref('')
       const caption = ref(initialNode.attrs.caption)
       const ratio = ref(initialNode.attrs.ratio)
       const selected = ref(false)
diff --git a/src/image-inline/view.ts b/src/image-inline/view.ts
index 6290114..bec0981 100644
--- a/src/image-inline/view.ts
+++ b/src/image-inline/view.ts
@@ -14,7 +14,7 @@ export const inlineImageView = $view(
   imageSchema.node,
   (ctx): NodeViewConstructor => {
     return (initialNode, view, getPos) => {
-      const src = ref(initialNode.attrs.src)
+      const src = ref('')
       const alt = ref(initialNode.attrs.alt)
       const title = ref(initialNode.attrs.title)
       const selected = ref(false)
diff --git a/lib/image-block/index.js b/lib/image-block/index.js
index ea1abe9..f898d4f 100644
--- a/lib/image-block/index.js
+++ b/lib/image-block/index.js
@@ -569,7 +569,7 @@ const imageBlockView = $view(
   imageBlockSchema.node,
   (ctx) => {
     return (initialNode, view, getPos) => {
-      const src = ref(initialNode.attrs.src);
+      const src = ref("");
       const caption = ref(initialNode.attrs.caption);
       const ratio = ref(initialNode.attrs.ratio);
       const selected = ref(false);
diff --git a/lib/image-inline/index.js b/lib/image-inline/index.js
index 3a83b6a..fdd3337 100644
--- a/lib/image-inline/index.js
+++ b/lib/image-inline/index.js
@@ -287,7 +287,7 @@ const inlineImageView = $view(
   imageSchema.node,
   (ctx) => {
     return (initialNode, view, getPos) => {
-      const src = ref(initialNode.attrs.src);
+      const src = ref("");
       const alt = ref(initialNode.attrs.alt);
       const title = ref(initialNode.attrs.title);
       const selected = ref(false);
